src/IK/src/InverseKinematics.cpp

```diff
@@ -0,0 +1,140 @@
+#include <BiomechanicalAnalysis/IK/InverseKinematics.h>
+#include <iostream>
+
+using namespace BiomechanicalAnalysis::IK;
+
+
+bool HumanIK::initialize(std::weak_ptr<const BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
+                std::shared_ptr<iDynTree::KinDynComputations> kinDyn)
+{
+    constexpr std::size_t highPriority = 0;
+    constexpr std::size_t lowPriority = 1;
+
+    m_jointPositions.resize(kinDyn->getNrOfDegreesOfFreedom());
+    m_jointVelocities.resize(kinDyn->getNrOfDegreesOfFreedom());
+
+    std::cout << "nr of dofs = " << kinDyn->getNrOfDegreesOfFreedom() << std::endl;
```

Comment:
we could use the logging functionalities implemented in https://github.com/ami-iit/biomechanical-analysis-framework/tree/main/src/Logging instead of printing with `cout`

Reply:
I deleted this `cout` because it is not very useful, I added it just for debugging.